index.html

```diff
@@ -1040,6 +1042,12 @@
 
 [=Cross-context recognition=] is only [=appropriate=] when the person being [=recognized=]
 can reasonably expect that recognition to happen and can control whether it does.
+Note that a person can use a piece of identifying information in two different
+contexts (e.g. their email or phone number) without that implying that they're
+using the same identity in both contexts. Unless there's some other indication
+that they intended to use a single identity, it is [=inappropriate=] to
+[=recognize=] them using that information.
+
```

Comment:
It seems like the inappropriateness is not just recognizing, but requesting specific cross-origin PII for the purpose of recognition.

Maybe:
> unless ..., it is inappropriate to seek identifying information for cross-context recognition, or to recognize them using such information.

Reply:
That sounds good, thanks.